apps/loom/loom.yml

```diff
@@ -0,0 +1,14 @@
+name: Loom
+description: 'Instantly shareable screen and camera recordings'
```

Comment:
Add a period before you close the quotes on `line 2`
Pretty common mistake.

Reply:
fixed